pkgs/development/python-modules/wrf-python/default.nix

```diff
@@ -0,0 +1,29 @@
+{lib, fetchPypi, python, pythonOlder, buildPythonPackage, gfortran, setuptools, mock, xarray, wrapt, numpy, fetchpatch}:
+
+buildPythonPackage rec {
+  pname = "wrf-python";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0maphsqfnrqpp52zz7xwcfjhw7y0fvq81mz91a9rfqvw0b2ww83m";
+  };
+  propagatedBuildInputs = [
+    gfortran
```

Comment:
Is `gfortran` needed for running the package? If is only a build requirement https://github.com/NCAR/wrf-python/blob/master/setup.py#L25-L45 then it should be specified in `buildInputs = [ ]`.